chart/kubeapps/templates/apprepository/apprepositories.yaml

```diff
@@ -60,5 +60,9 @@ spec:
         name: {{ printf "apprepo-%s-secrets" .name }}
     {{- end }}
   {{- end }}
+  {{- if .filterRule }}
+  filterRule:
+    {{- toYaml .filterRule | nindent 4 }}
+  {{- end }}
```

Comment:
Just an internal node, it would require an increase in the patch version of the chart, but we will do when syncing with the official Bitnami repo.